application.go

```diff
@@ -353,6 +353,19 @@ func (r *marathonClient) Application(name string) (*Application, error) {
 	return wrapper.Application, nil
 }
 
+// ApplicationConfiguration retrieves the application configuration from marathon
+// 		name: 		the id used to identify the application
+// 		version:  the version of the configuration you would like to receive
+func (r *marathonClient) ApplicationConfiguration(name, version string) (*Application, error) {
```

Comment:
I'd rename this to `ApplicationByVersion` in order to be both more consistent with `Application` (that already retrieves a configuration and doesn't have the `Configuration` suffix) and distinguish from `ApplicationVersions` (which doesn't retrieve any configurations but just a list of versions). I think the Marathon naming is a bit unfortunate and ambiguous, so we should try to do better. :-)